modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java

```diff
@@ -880,6 +894,8 @@ public void testPostAccountUserAccountsByExternalReferenceCodeByEmailAddress()
 	public void testPostUserAccount() throws Exception {
 		super.testPostUserAccount();
 
+		_testPostUserAccountWithMoreExternalReferenceCodes();
```

Comment:
at the end of the test

Reply:
@smottal does not work, I get 403 access denied error due to the code coming after it.

Reply:
it seems it is happening for post only